plutarch-ledger-api/src/Plutarch/LedgerApi/V1/Contexts.hs

```diff
@@ -0,0 +1,59 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
```

Comment:
I don't see any orphan here

Reply:
Any `PConstantDecl` instance is an orphan, and we create an instance of it for `ScriptContext`.